app/src/main/res/layout/item_title.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
check please org.p2p.wallet.home.ui.main.adapter.HeaderViewHolder and reuse it if we can
i see SemiBold.Title3 and SemiBold.Text1
but i presume it should be same and we need to force our designers to reuse headers